src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -89,6 +90,7 @@ def post(self, request: Request) -> Response:
         if request.query_params.get("dummyOccurrence") == "True":
             occurrence = {
                 "id": "55f1419e73884cd2b45c79918f4b6dc5",
+                "project_id": 1,
```

Comment:
we should use the same project id as the event that we get from the user account